tornado/web.py

```diff
@@ -1786,8 +1787,8 @@ def __init__(self, handlers=None, default_host="", transforms=None,
                 self.transforms.append(GZipContentEncoding)
         else:
             self.transforms = transforms
-        self.handlers = []
-        self.named_handlers = {}
+        self.default_router = RegexRouter(self)
+        self.routers = [self.default_router]
```

Comment:
What's the use case for multiple routers per application? I was thinking there would be one router per application.